src/common/tse.c

```diff
@@ -367,15 +369,22 @@ static void
 tse_task_complete_locked(struct tse_task_private *dtp,
 			 struct tse_sched_private *dsp)
 {
-	if (dtp->dtp_completed)
+	D_INFO("EJMM tse.c: tse_task_complete_locked()");
+
+	if (dtp->dtp_completed) {
+ 		D_INFO("EJMM tse.c: dtp->dtp_completed");
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs